cob_bringup_sim/package.xml

```diff
@@ -20,6 +20,7 @@
   <exec_depend>gazebo_msgs</exec_depend>
   <exec_depend>gazebo_ros</exec_depend>
   <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>roslib</exec_depend>
```

Comment:
Hasn't this been done in https://github.com/ipa320/cob_simulation/commit/ec32055b7d057d70994e21752a89fcc2593944b1 already?
Please pull ipa320/indigo_dev or rebase...